cycode/cli/files_collector/sca/base_restore_dependencies.py

```diff
@@ -24,10 +25,12 @@ def execute_command(command: List[str], file_name: str, command_timeout: int) ->
 
 
 class BaseRestoreDependencies(ABC):
-    def __init__(self, context: click.Context, is_git_diff: bool, command_timeout: int) -> None:
+    def __init__(self, context: click.Context, is_git_diff: bool, command_timeout: int,
+                 create_output_file_manually: bool) -> None:
```

Comment:
let's add default value to `create_output_file_manually` here. False one. and remove all `OUTPUT_FILE_MANUALLY = False` in the code